### PR TITLE
Add Mastercard 3DS card (take #2)

### PR DESCRIPTION
**Pull request by sam-artuso**
Reference: https://docs.adyen.com/development-resources/testing/test-card-numbers/#enrolled-in-3d-secure-2

### CODE REVIEW

File: data/cards.json [hunk 1]
```diff
@@ -279,11 +279,18 @@
                 "expiry" : "03/30",
                 "CVC" : "737",
                 "country" : "BR"
-            }
+            },
+            {
+              "cardNumber" : "5454 5454 5454 5454",
```
Comment (gcatanese) [inline data/cards.json:284]: It should be `cardnumber`

**Review by gcatanese — CHANGES_REQUESTED** [review]
@sam-artuso Thanks for the PR. There is a small change required, we are then ready to merge. Next week we plan to release the new version.

Please remove the `WIP` tag when this is ready

At the time of branching the E2E tests on `main` were incorrect, we have now addressed it (#43 ).

**Comment by sam-artuso** [comment]
Rebased on top of master and squashed my commits.

**Comment by gcatanese** [comment]
Looks great, thank you!

**Review by Spacebounddaily — APPROVED** [review]

**Pull request merged**